alphadia/libtransform.py

```diff
@@ -328,7 +328,7 @@ def forward(self, input: SpecLibBase) -> SpecLibBase:
             logging.info(f"Loading PeptDeep models of type {self.peptdeep_model_type}")
             model_mgr.load_installed_models(self.peptdeep_model_type)
 
-        if self.peptdeep_model_path is not None:
+        if self.peptdeep_model_path is not None and self.peptdeep_model_path != "":
```

Comment:
`if self.peptdeep_model_path`?

Reply:
should we run e2e tests here?
(classic 'submarine' PR: disguised as UI change, but changes default parameters ;-))

Reply:
Yes, to some degreee :D But all of these parameters are redefined in the e2e tests. So we only change to enable Oscars optimization by default.